src/plugins/intel_cpu/src/nodes/executors/executor_factory.hpp

```diff
@@ -284,11 +162,11 @@ class ExecutorFactory {
     std::vector<ExecutorPtr> m_executors;
```

Comment:
Seems like `m_implementationRequiresFallback` and `m_executors` fields should be removed

Reply:
My bad, removed